frontend/express/public/javascripts/countly/vue/templates/sidebar/users-menu.html

```diff
@@ -0,0 +1,198 @@
+<el-popover
```

Comment:
As I recall, el-popover was a leaky component. So I avoided using it. I'll double check.